xarray/tests/test_backends.py

```diff
@@ -2278,10 +2279,13 @@ def create_zarr_target(self):
         raise NotImplementedError
 
     @contextlib.contextmanager
-    def create_store(self):
+    def create_store(self, cache_members: bool = True):
```

Comment:
```suggestion
    def create_store(self):
```

I would just let the default handle this

Reply:
I added this kwarg because the cache has to be turned off specifically for the round-trip test that re-uses the `ZarrStore` instance, but I have since addressed that by overriding that test specifically